reference/ToDo/src/ToDo/Styles/NavigationBar.xaml

```diff
@@ -1,25 +1,24 @@
-﻿<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-    xmlns:utu="using:Uno.Toolkit.UI">
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                    xmlns:utu="using:Uno.Toolkit.UI">
 
-    <Style
-        x:Key="RootModalMainCommandStyle"
-        BasedOn="{StaticResource ModalMainCommandStyle}"
-        TargetType="AppBarButton">
-        <Setter Property="Foreground" Value="{ThemeResource PrimaryBrush}" />
-        <Setter Property="Icon">
-            <Setter.Value>
-                <BitmapIcon UriSource="ms-appx:///Assets/Icons/close.png" />
-            </Setter.Value>
-        </Setter>
-    </Style>
+  <Style x:Key="RootModalMainCommandStyle"
+         BasedOn="{StaticResource ModalMainCommandStyle}"
+         TargetType="AppBarButton">
+    <Setter Property="Foreground"
+            Value="{ThemeResource OnSurfaceBrush}" />
+    <Setter Property="Icon">
+      <Setter.Value>
+        <BitmapIcon UriSource="ms-appx:///Assets/Icons/close.png" />
+      </Setter.Value>
+    </Setter>
+  </Style>
 
-    <Style
-        x:Key="RootModalNavigationBarStyle"
-        BasedOn="{StaticResource ModalNavigationBarStyle}"
-        TargetType="utu:NavigationBar">
-        <Setter Property="MainCommandStyle" Value="{StaticResource RootModalMainCommandStyle}" />
-    </Style>
+  <Style x:Key="RootModalNavigationBarStyle"
+         BasedOn="{StaticResource ModalNavigationBarStyle}"
+         TargetType="utu:NavigationBar">
+    <Setter Property="MainCommandStyle"
+            Value="{StaticResource RootModalMainCommandStyle}" />
+  </Style>
```

Comment:
We should eventually remove these weird "modal" styles. We can get away with just changing background/foreground

Reply:
Yeah I will revise that, thanks

Reply:
Removed